tests/ui/crate_level_checks/no_std_swap.rs

```diff
@@ -6,7 +6,7 @@ use core::panic::PanicInfo;
 
 #[warn(clippy::all)]
 fn main() {
-    // TODO: do somethjing with swap
+    // TODO: do something with swap
```

Comment:
Not sure what this comment is doing there in the first place. I guess this could be removed completely.

Reply:
Whoops yeah, I'd forgotten to remove this. I think we should remove this completely. Thanks for this!